deploy/examples/csi-operator.yaml

```diff
@@ -14483,13 +14474,73 @@ spec:
 ---
 apiVersion: v1
 kind: ServiceAccount
+metadata:
+  name: ceph-csi-operator-cephfs-ctrlplugin-sa
```

Comment:
Do we plan to use these CSI serviceaccount names for Rook? i though we wanted to have csi-cephfs-ctrlplugin-sa by default https://github.com/ceph/ceph-csi-operator/issues/106#issuecomment-2273898234

Reply:
I wanted to do minimum manual changes in what is generated by `make build-installer` but if we want I can do the change @travisn.

Reply:
I see, then I'll update after @travisn comment

Reply:
Yes, can we shorten these names? Does that mean when generating them, you just need to set `NAME_PREFIX` to blank? Hopefully that keeps it simple in your conversion script and doesn't need manual steps.

Reply:
we just need to run as below
```bash
NAME_PREFIX=ceph-csi- NAMESPACE=rook make build-csi-rbac
```

all the namewill have ceph-csi- prefixed and rook namespace, below is one example

```yaml
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRoleBinding
metadata:
  name: ceph-csi-rbd-ctrlplugin-crb
roleRef:
  apiGroup: rbac.authorization.k8s.io
  kind: ClusterRole
  name: ceph-csi-rbd-ctrlplugin-cr
subjects:
- kind: ServiceAccount
  name: ceph-csi-rbd-ctrlplugin-sa
  namespace: rook
---
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRoleBinding
metadata:
  name: ceph-csi-rbd-nodeplugin-crb
roleRef:
  apiGroup: rbac.authorization.k8s.io
  kind: ClusterRole
  name: ceph-csi-rbd-nodeplugin-cr
subjects:
- kind: ServiceAccount
  name: ceph-csi-rbd-nodeplugin-sa
  namespace: rook
```

Reply:
updated